detect_secrets/core/audit.py

```diff
@@ -283,13 +290,13 @@ def determine_audit_results(baseline, baseline_path):
         audit_results['stats'][audit_result]['count'] += 1
         audit_results['stats'][audit_result]['files'][filename] += 1
         total += 1
-    audit_results['stats']['signal'] = str(
+    audit_results['stats']['signal'] = '{:.2f}'.format(
```

Comment:
The `%` go into this format string

Reply:
Great catch!! I no longer need the `[:4]` so that makes a lot of sense 🎊